lance-spark-base_2.12/src/main/scala/org/apache/spark/sql/util/LanceArrowUtils.scala

```diff
@@ -37,6 +37,8 @@ import java.util.concurrent.atomic.AtomicInteger
 import scala.collection.JavaConverters._
 
 object LanceArrowUtils {
+  val ARROW_FIXED_SIZE_LIST_SIZE_KEY = "arrow.FixedSizeList.size"
```

Comment:
is this an existing parameter, or something new? Feels strange to have carmel case here, if we can change it it should use something like `arrow.fixed-size-list.size` or `arrow_fixed_size_list_size`

Reply:
it's a new parameter added for identify which column needs conversion, OKKK can do arrow.fixed-size-list.size or arrow_fixed_size_list_size

Reply:
changed to arrow.fixed-size-list.size